README.md

```diff
@@ -9,5 +9,67 @@ An opinionated mochitest wrapper for running mochitests and reporting the result
 
 ```bash
 npm i -g mochii
-mochii
 ```
+
+#### Mochii Runner
+
+Mochii tries to be a thin wrapper around `mochitest`, with some extra configuration.
+
+```bash
+mochii browser_dbg-expressions.js
+mochii -- --jsdebugger browser_dbg-expressions
+```
+
+Mochii supports some options that will hopefully save you some keystrokes:
+
+* *mozilla-central* - path to the repo you want to run the tests
+* *test-path* - path to the tests you want to run
+
+We recommend creating a bash function or an npm script to save the settings:
+
+##### Bash Function
+
+```bash
+function mochi() {
+  mochii --mc foo --default-test-path yo -- $1 $2 $3 $4
+}
+```
+
+##### NPM scripts
+
+```bash
+{
+  scripts: {
+    mochi: "mochii --mc ../mozilla-central --default-test-path debugger/client/debugger/new --",
+    mochid: "yarn mochi -- --jsdebugger -- ",
+    mochir: "yarn mochi -- --repeat 10 -- ",
+    mochih: "yarn mochi -- --set-env=HEADLESS -- "
+  }
+}
+```
+
+Our new `mochi` script is now setup to run the mochitests in the `../mozilla-central` with
```

Comment:
`mochii`?

Reply:
lol - i named the function above mochi. it's super confusing